cli/cmdlineparser.cpp

```diff
@@ -1837,9 +1837,10 @@ void CmdLineParser::printHelp(bool premium) const
         "                         this is not needed.\n"
         "    --include=<file>\n"
         "                         Force inclusion of a file before the checked file.\n"
-        "    -i <str>             Exclude source files or directories matching str from\n"
-        "                         the check. This applies only to source files so header\n"
-        "                         files included by source files are not matched.\n"
+        "    -i <str>             Skip translation units whose *source filename* matches\n"
```

Comment:
Presumably `-i` is derived from 'ignore', so maybe that should be used in the help text?